code/_core/datum/plant/_plant_type.dm

```diff
@@ -246,4 +246,72 @@
 		/reagent/nutrition/lipolicide = 0.25,
 		/reagent/toxin/angel_toxin = 0.5,
 		/reagent/drug/space = 0.25
-	)
\ No newline at end of file
+	)
+
+/plant_type/corn
+	name = "corn"
+	desc = "Corn is the smartest plant alive. It evolved Space Americans to grow it."
+	plant_icon_state = "corn"
+	harvest_icon_state = "corn"
+	seed_icon_state = "corn"
+	plant_icon_count = 1
+	reagents = list(
+		/reagent/nutrition/corn_flour = 1,
```

Comment:
I think this should just create corn as a nutrient instead of just corn flour. Corn flour should be able to be made from the corn reagent if it's ground up. See ```/reagent/nutrition/wheat_grain``` in reagent_nutrition_grain.dm on how to do this.